app/components/directives/address.js

```diff
@@ -0,0 +1,14 @@
+(
+    function(){
+        angular.module('MockApp')
+            .component('address',{
+                template:
+                    '<b>Street</b> : {{$ctrl.model.street}}<br/> '+
+                    '<b>City</b>: {{$ctrl.model.city}}<br/>'+
+                    '<b>Zipcode</b>: {{$ctrl.model.zipcode}} ',
+                bindings:{
+                    model: '=',
```

Comment:
What does model represent here? This variable name isn't more informative.

Reply:
Shall I change the model to value?

Reply:
It should be more informative. Change this to what it should represent